report/web.py

```diff
@@ -106,10 +109,17 @@ def get_generated_targets(benchmark: str) -> list[str]:
 
 
 def list_benchmarks() -> List[Benchmark]:
+  """Lists benchmarks in the result directory."""
   benchmarks = []
   benchmark_names = sorted(os.listdir(RESULTS_DIR))
-  for benchmark in benchmark_names:
-    results, targets = get_results(benchmark)
+  # Not sure why there is a `lost+found` dir in |RESULTS_DIR|, which caused
+  # failures in get_generated_targets().
+  # Maybe it is from mounting?
+  # TODO(erfan): Check if not mounting to the root dir can solve this.
+  benchmark_names = sorted(
+      [dir for dir in os.listdir(RESULTS_DIR) if dir != 'lost+found'])
```

Comment:
@erfanio : I started to observe this error after mounting `results-volume`.
Could we avoid this error by mounting a different directory (or other ways)?
Thanks :)

Reply:
Looks like this is usually due to some disk corruption. 

Is there some other way we can filter this out, without resorting to hardcoding this directory name? 

e.g. ensuring that the directory contains some file/structure that matches what we expect from a benchmark, or just continuing when we hit an exception below (with a log).

Reply:
> just continuing when we hit an exception below (with a log)

Yep, I can do that!

> ensuring that the directory contains some file/structure that matches what we expect from a benchmark

I am not quite sure what the content of `lost+found` is; hence, I cannot tell what must be in a benchmark dir and not in it.

But continuing with a log is a quick fix I can make.

Reply:
Fixed by adding an overall validation, because there are too many contents in the benchmark dir to check individually.